src/streamdiffusion/acceleration/tensorrt/controlnet_models.py

```diff
@@ -0,0 +1,207 @@
+"""
+ControlNet TensorRT Model Definitions
+
+This module provides TensorRT model definitions for ControlNet compilation,
+following the same patterns as the base UNet models but optimized for
+ControlNet-specific inputs and outputs.
+"""
+
+from typing import List, Dict, Optional
+from .models import BaseModel
+import torch
+
+
+class ControlNetTRT(BaseModel):
+    """TensorRT model definition for ControlNet compilation"""
+    
+    def __init__(self, 
+                 controlnet_type: str = "canny",
+                 fp16: bool = True,
+                 device: str = "cuda",
+                 max_batch: int = 2,
+                 min_batch_size: int = 1,
```

Comment:
@ryanontheinside Can this be named `max_batch_size` or rename `min_batch_size` to `min_batch` to keep the param names consistent? Curious what led to this change, is there a conflict with another param downstream?